changelogs/7.4.asciidoc

```diff
@@ -28,3 +28,4 @@ https://github.com/elastic/apm-server/compare/v7.3.1\...v7.4.0[View commits]
 - Add support for host.name, adapt Intake API to accept detected and configured host information {pull}2531[2531], {pull}2540[2540].
 - Index `http.request.socket.remote_address` as `client.ip` {pull}2588[2588].
 - Upgrade Go to 1.12.9 {pull}2635[2635].
+- Make metricset timestamp optional {pull}2711[2711].
```

Comment:
you could even say this is a bug

Reply:
Yeah I was in between on where to put it. However, so far all agents sending metricsets sent up the timestamp, so it is now loosening the requirements for a new integration with the RUM agent, and therefore I decided with not labeling it as a bug. If you feel strongly I am not against moving it to the bug section though.